docker-compose.test.yml

```diff
@@ -105,7 +105,8 @@ services:
         KERAS_PACKAGE: git+https://github.com/keras-team/keras.git
         PYTORCH_PACKAGE: torch-nightly
         PYSPARK_PACKAGE: pyspark==2.4.0
-  test-cpu-openmpi-py3_6-tfhead-kerashead-torchhead-mxnet1_4_0-pyspark2_4_0:
+        MXNET_PACKAGE: mxnet --pre
```

Comment:
why not use the same 1.5.0b20190412 as above here?

Reply:
This is the nightly test. We want to test our nightly package instead of a specific release.